packages/super-components/components/msc-network-tile/src/msc-network-tile/components/msc-network-vrack.tsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable @typescript-eslint/no-inferrable-types */
+import { Component, h, Host, Prop, State } from '@stencil/core';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components/text';
+import { apiClient } from '@ovh-ux/manager-core-api';
+
+import {
+  Translations,
+  VrackInfos,
+  IpLoadbalancingList,
+  IpsList,
+  NatIpInfos,
+} from '../msc-network-types';
+
+export interface IMscNetworkVrack {
+  localeStrings: Translations;
+}
+
+@Component({
+  tag: 'msc-network-vrack',
+  styleUrl: 'msc-network-components.scss',
+  shadow: true,
+})
+export class MscNetworkVrack implements IMscNetworkVrack {
+  @Prop() localeStrings: Translations;
+
+  @Prop() public serviceName = '';
+
+  @Prop() ipName = '';
+
+  @Prop() natIp = '';
+
+  @State() private vrackInfos?: VrackInfos;
+
+  @State() private ipLoadbalancingList?: IpLoadbalancingList;
+
+  @State() private ipsList: IpsList;
+
+  @State() private ipFOState: string = '';
+
+  @State() private natIpInfos?: NatIpInfos;
```

Comment:
Is there any added value to use type alias instead of `string[]` in these cases ?

Reply:
Ok for the NatIpInfos
But what about VrackInfos, IpLoadbalancingList and IpsList ?